app/views/paymentOptions.html

```diff
@@ -0,0 +1,18 @@
+<div class="sub-page-title-block">
```

Comment:
So....even if they've just searched for their ticket and clicked payment options, we're going to make them enter their DOB again? It's fine for now but seems like we should be able to track whether they just came from a search result or from a link in SMS from a security perspective.